src/protocols/ssh/ssh.c

```diff
@@ -256,6 +256,10 @@ void* ssh_client_thread(void* data) {
         return NULL;
     }
 
+    /* Set the client timezone */
+    if (settings->timezone != NULL)
+        libssh2_channel_setenv(ssh_client->term_channel, "TZ", settings->timezone);
```

Comment:
Same here - a warning should be logged if this cannot be set.

Reply:
Done.